.github/workflows/label-analysis.yml

```diff
@@ -102,25 +102,24 @@ jobs:
     if: github.triggering_actor != 'dd-devflow[bot]'
     runs-on: ubuntu-latest
     steps:
+      - name: Checkout repository
+        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+        with:
+          persist-credentials: false
+      - name: Setup Python3
+        uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5.3.0
+        with:
+          python-version: "3.12.6"
+          cache: "pip"
+          cache-dependency-path: '**/requirements*.txt'
+      - name: Install python dependencies
+        run: pip3 install -r requirements.txt
       - name: Check qa/[done|no-code-change] labels are not set together
-        run: |
-          is_qa_done=1
-          is_qa_no_code_change=1
-          for label in $LABELS; do
-            if [[ "$label" == "qa/done" ]]; then
-              is_qa_done=0
-            fi
-            if [[ "$label" == "qa/no-code-change" ]]; then
-              is_qa_no_code_change=0
-            fi
-          done
-          if [ $is_qa_done -eq 0 ] && [ $is_qa_no_code_change -eq 0 ]; then
-            echo "Both 'qa/done' and 'qa/no-code-change' labels are set -- only one of them should be set"
-            exit 1
-          fi
-          echo "No issue with 'qa/done' and 'qa/no-code-change' labels"
         env:
           LABELS: ${{ needs.fetch-labels.outputs.LABELS}}
+        run: |
+          invoke -e github.check-qa-labels -l "${LABELS[@]}"
```

Comment:
```suggestion
          inv -e github.check-qa-labels -labels "${LABELS[@]}"
```
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I'd recommend to use the full argument name because it's not always obvious how invoke reduces the arguments (when there are several starting with an `l` for example)